scripts/integration/docker-compose.axiom.yml

```diff
@@ -51,10 +51,12 @@ services:
       - axiom-db
     volumes:
       - ${PWD}:/code
+      - target:/code/target
```

Comment:
Per https://github.com/vectordotdev/vector/pull/12977 we don't actually want to mount `target/` as it causes issues with permissions with the host system.

Reply:
It's making a `volume` not mounting from the host though?

Reply:
😕 ❓  Oh I added it to these files after the prior thread here: https://github.com/vectordotdev/vector/pull/13793#discussion_r936900800

, that issue #12977 , the code changes look to me like the target is actually being added, not removed? What am I missing here ...

Reply:
Doh, yes, ignore me 😅  This is creating a new volume for `target` as we desire.